src/main/java/com/myorg/mvc/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.myorg.mvc.exceptions;
+
+public class UserNotFoundException extends RuntimeException {
```

Comment:
Please create service layer exception and extends all other exceptions from service layer from it.